spec/requests/feedback_messages_spec.rb

```diff
@@ -68,8 +68,8 @@ def mock_recaptcha_config_enabled
         expect(FeedbackMessage.where(offender_id: user.id).count).to eq(1)
       end
     end
-
-    context "with no recaptcha keys set" do
+    
```

Comment:
**nitpick (blocking):** Please clean up this whitespace change.

Reply:
@citizen428 These changes are done